app/javascript/gobierto_dashboards/modules/contracts_controller.js

```diff
@@ -103,6 +103,8 @@ export class ContractsController {
         EventBus.$once('moved_to_summary', () => {
           this._redrawCharts();
         });
+
+        document.getElementsByClassName("js-loading")[0].classList.add('hidden');
```

Comment:
Puff, disgusting.
```suggestion
        const elem = document.querySelector(".js-loading");
        if (elem) {
          elem.classList.add('hidden')
        }
```

If you need more than one, `querySelectorAll` gives you a `NodeList` (iterable)